src/sparse/impl/KokkosSparse_spgemm_mkl2phase_impl.hpp

```diff
@@ -529,6 +533,10 @@ void mkl2phase_symbolic(
       if (SPARSE_STATUS_SUCCESS != mkl_sparse_destroy (C)){
         throw std::runtime_error ("Error at mkl_sparse_destroy C\n");
       }
+#elif __INTEL_MKL__ == 2018 && __INTEL_MKL_UPDATE__ < 2
+      throw std::runtime_error ("MKL version 18 must have update 2 - use intel/18.2.xyz compiler\n");
```

Comment:
Minor nit-pick - pls remove the word compiler and instead say "Intel MKL version 18.2. or more"

Reply:
Fixed